src/compiler/scala/tools/nsc/typechecker/Contexts.scala

```diff
@@ -1244,9 +1228,7 @@ trait Contexts { self: Analyzer =>
       lookupScopeEntry(original) match {
```

Comment:
the comment just above this line is now out of sync

Reply:
@lrytz I've fixed the comments.